### PR TITLE
[HAMMER] Add hammer translations

### PR DESCRIPTION
**Pull request by mzazrivec**

### CODE REVIEW

File: client/gettext/json/available_languages.json [hunk 1]
```diff
@@ -1 +1 @@
-{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
+{"es-ES":"Español","es":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```
Comment (AllenBW) [inline client/gettext/json/available_languages.json]: @mzazrivec tests are thinking this ought to look like this...  Español being in there twice looks to be a woopsie 🤔 
```suggestion
{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```

File: client/gettext/json/available_languages.json [hunk 2]
```diff
@@ -1 +1 @@
-{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
+{"es-ES":"Español (España)","es":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```
Comment (AllenBW) [inline client/gettext/json/available_languages.json]: @mzazrivec travis still hates us, if string is indeed correct, the test ought to be updated https://github.com/ManageIQ/manageiq-ui-service/blob/ae385823137b78b78b1ea6b8abdee77b0b9b2ac4/client/app/core/language.service.spec.js#L78

Comment (mzazrivec) [reply client/gettext/json/available_languages.json]: We have `es-ES` translation (not complete) and also `es` translation (100% complete). For product, we'll be enabling just `es`, I'm sure.

I wasn't aware of that test or what it does, though I can fix it to say ` expect(locale).to.eq('es')`

Comment (AllenBW) [reply client/gettext/json/available_languages.json]: yeah!! that should do it! 👍

**Comment by AllenBW** [comment]
we're g2g here @simaishi !! Thanks for the spec fix @mzazrivec 🤗

**Pull request merged**